README.md

```diff
@@ -1082,6 +1082,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Free Hosting](https://freehostingnoads.net/) — Free Hosting With PHP 5, Perl, CGI, MySQL, FTP, File Manager, POP E-Mail, free sub-domains, free domain hosting, DNS Zone Editor, Web Site Statistics, FREE Online Support and many more features not offered by other free hosts.
   * [Freehostia](https://www.freehostia.com) — FreeHostia offers free hosting services incl. an industry-best Control Panel & a 1-click installation of 50+ free apps. Instant setup. No forced ads.
   * [Neocities](https://neocities.org) — Static, 1 GB free storage with 200 GB Bandwidth.
+  * [Deno Deploy](https://deno.com/deploy) - Serverless JavaScript hosting with zero config, worldwide powered by deno.
```

Comment:
It's already in the list, thank you for the contribution though!

Reply:
Thanks!!